packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1553,7 +1553,7 @@ async function handler(
   async function createBooking() {
     if (originalRescheduledBooking) {
       evt.title = originalRescheduledBooking?.title || evt.title;
-      evt.description = originalRescheduledBooking?.description || evt.additionalNotes;
+      evt.description = originalRescheduledBooking?.description || evt.description;
```

Comment:
To me, it seems like the right change.

But this is a change in the core flow and needs confirmation from @zomars @emrysal. Blocking merge till then.

Reply:
any update?

Reply:
@hariombalhara @alannnc is this safe to merge?

Reply:
I think back then was some kind of error and description was arriving on additionalNotes? Something like that... I believe this is the right thing to do now so Safe to merge. @PeerRich